development/components/form/form-theme/form-theme.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Symfony Form Theme for PrestaShop
+weight: 10
+---
+
+# Symfony Form Theme for PrestaShop
+
+## PrestaShop usage
+
+PrestaShop relies on Twig's FormRenderer, which requires a **Form Theme** to function. [Bootstrap 4 Form Theme][sf-bootstrap4-form-theme] is a popular one.
+
+The Form Theme is a set of Twig macros and functions that provide a way to render the different parts of a form: every label, input type, and specific option has its own macro.
+
+Example for the date widget:
+
+```twig
+{%- block date_widget -%}
+  {%- if widget == 'single_text' -%}
+    {{ block('form_widget_simple') }}
+  {%- else -%}
+    <div {{ block('widget_container_attributes') }}>
+      {{- date_pattern|replace({
+        '{{ year }}':  form_widget(form.year),
+        '{{ month }}': form_widget(form.month),
+        '{{ day }}':   form_widget(form.day),
+      })|raw -}}
+    </div>
+  {%- endif -%}
+{%- endblock date_widget -%}
+```
+
+## PrestaShop form themes
+
+PrestaShop Form themes are located in the `src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/` directory.
+
+### Original PrestaShop Form theme
+
+{{% notice warning %}}
+**This theme is limited and deprecated.**
+{{% /notice %}}
+
+Originally, Symfony forms in PrestaShop were developed using the field-by-field rendering technique, where each field is rendered individually ([see example](https://github.com/PrestaShop/PrestaShop/blob/1.7.7.0/src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/employee_options.html.twig)). This method allows a fine-grained control of the rendering, but limits the extension capacities, which is why
+it is progressively being replaced by a new one (read below).
+
+This form theme is a copy of Symfony's original Bootstrap 4 form theme, with customizations made for PrestaShop.
+
+Its base theme is `bootstrap_4_layout.html.twig`. Used to render Symfony forms vertically, it relies on multiple files:
+
+- form_div_layout
+- typeahead
+- material
+
+By default, PrestaShop's Twig templates will use `bootstrap_4_horizontal_layout.html.twig` (as configured in PrestaShop's `/app/config/config.yml` file). This child of the above theme render forms horizontally instead of vertically.
+
+{{< figure src="../../../img/old-form-theme-17.png" title="Original Form Theme example of rendering" >}}
```

Comment:
Consider moving the images into the directory of the page that uses them, to avoid path traversal
```suggestion
{{< figure src="../../../img/old-form-theme-17.png" title="Original Form Theme example of rendering" >}}
```